packages/components/src/components/video-player/video-player.scss

```diff
@@ -1,6 +1,7 @@
 .player {
     &__wrapper {
         position: relative;
+        background-color: var(--general-main-1);
```

Comment:
It will affect other places, where Video is used (wallets, Dtrader V1). May we it can be only for V2? By passing className

Reply:
v2 only changes now